layouts/partials/footer.html

```diff
@@ -11,7 +11,7 @@
 			</div>
 		</section>
 
-    <script async src="/js/main.js"></script>
+    <script async src="{{ .Site.BaseURL }}js/main.js"></script>
```

Comment:
Wouldn't it make more sense to use `"{{ .Site.BaseURL }}/js/main.js"` with an additional slash here?